dev-php/pecl-parallel/pecl-parallel-1.1.4.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+PHP_EXT_NAME="parallel"
+PHP_EXT_INI="yes"
+PHP_EXT_ZENDEXT="no"
+
+USE_PHP="php7-3 php7-4"
+
+inherit php-ext-pecl-r3
+
+SRC_URI="${SRC_URI} -> ${P}.tgz"
```

Comment:
Yeah let's replicate the eclass URL here.

I doubt the eclass will **ever** drop or modify that URL, but just in case it does...